### DISCUSSION TITLE
Window still has rounded corners even with window-decoration = false

### DISCUSSION THREAD

**Discussion by ndavd** [ghostty-org/ghostty#3924] [Feature Requests, Ideas] [answered]
I'm on Linux, BSPWM. With `window-decoration = false`, I still get rounded corners, and with `gtk-adwaita = false`, only the bottom ones get fixed.

Similar discussion: https://github.com/ghostty-org/ghostty/discussions/3290.

**Comment by rhodes-b** [comment]
You should look into your WM applying any gtk styling to windows globally

**Comment by ndavd** [comment, marked as answer]
I'll close this as discussion is continued on https://github.com/ghostty-org/ghostty/discussions/3290.
